fdmi/source_dock_fom.c

```diff
@@ -837,6 +865,71 @@ static void fdmi_rr_fom_fini(struct m0_fom *fom)
 	M0_LEAVE();
 }
 
+static int m0_fdmi__handle_release(struct m0_uint128 *fdmi_rec_id)
```

Comment:
Minor style: maybe there is no need for `m0_` prefix in static function.

Reply:
OK. This piece of code of this function, is moved from source_dock.c to this file, unchanged.
In next phase, a new task is needed to address all the TODOs in this function.

Reply:
The piece of code is restored.
Improvement and optimization of this function will be scheduled in next PI.